### PR TITLE
backend/electrum: make Headers/GetMerkle rpc calls blocking/synchronous

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: backend/coins/btc/headers/headers.go [hunk 1]
```diff
@@ -433,6 +436,7 @@ func (headers *Headers) VerifiedHeaderByHeight(height int) (*wire.BlockHeader, e
 }
 
 func (headers *Headers) kick() {
+	time.Sleep(30 * time.Second)
```
Comment (Beerosagos) [inline backend/coins/btc/headers/headers.go:439]: Was this left here intentionally? In my test, it seems to slow down the sync, which takes forever.

File: backend/coins/btc/headers/headers.go [hunk 2]
```diff
@@ -180,7 +180,10 @@ func (headers *Headers) Initialize() {
 			headers.update(header.BlockHeight)
 		},
 	)
-	headers.kickChan <- struct{}{}
+	go func() {
```
Comment (Beerosagos) [inline backend/coins/btc/headers/headers.go:183]: Why do we need the 30s sleep?

File: backend/coins/btc/headers/headers.go (hunk 1 above)
Comment (benma) [reply backend/coins/btc/headers/headers.go]: omg no, thanks for finding it. quite embarrassing :sweat_smile:

File: backend/coins/btc/headers/headers.go (hunk 2 above)
Comment (benma) [reply backend/coins/btc/headers/headers.go]: we don't, it's a debug leftover :cold_sweat: thanks, undone!

**Review by Beerosagos — APPROVED** [review]
tACK!

**Pull request merged**